mobile/package.json

```diff
@@ -59,5 +59,10 @@
 			"!artifacts/**/*.spec.js",
 			"!artifacts/**/*.index.js"
 		]
+	},
+	"rnpm": {
```

Comment:
dafaq is this?

Reply:
react-native cli copies the assets to the projects from this path

Reply:
I've added it manually that rnpm copies it to the project

Reply:
this is what I'm asking. Why? all other fonts work without rnpm

Reply:
This is a custom font. To use it should be added to the project as resource. rnpm does it at once to both platforms. is there another way?